handsontable/src/tableView.js

```diff
@@ -250,6 +250,10 @@ class TableView {
     priv.table = rootDocument.createElement('TABLE');
     addClass(priv.table, 'htCore');
 
+    priv.table.ariaRowCount = this.settings?.data?.length;
+    priv.table.ariaColCount = this.settings?.data[0]?.length;
```

Comment:
I guess the values won't be updated after dataset length change. Is that is ok? Other question why not to use regular API such as `countRows`/`countCols`? I haven't run the code yet but it seems that it won't work for Array of Objects 🤔 

Can I ask to write tests for that aria tags?